deployment/postgres-init-db/sql/types/util_functions/request_status.sql

```diff
@@ -0,0 +1 @@
+create type util_functions.request_status as enum('pending', 'incomplete', 'failed', 'success');
```

Comment:
Thanks, @Mythicaeda , for clarifying for me that the need for a hasura path, described in this hasura ticket https://github.com/hasura/graphql-engine/issues/4630 , is related to, but ultimately independent of the decision to keep the two `request_status` types separate, or to merge them

Reply:
A consideration I had was that any change to this type will require writing a migration for both `simulation_dataset` and `scheduling_request`. That said, if we decide to add a variant for the scheduler only, I imagine we can at that point define a new enum `scheduler_request_status`, and write a migration only for the `scheduling_request` table. In that case, I think there's no harm in combining the two types now.